src/main/java/dawahdevs/qa/api/transformer/QaApiTransformer.java

```diff
@@ -0,0 +1,140 @@
+package dawahdevs.qa.api.transformer;
+
+import static dawahdevs.qa.api.util.QaApiUtil.optionalMap;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Component;
+
+import dawahdevs.qa.api.entity.DbAuthor;
+import dawahdevs.qa.api.entity.DbQuestion;
+import dawahdevs.qa.api.entity.DbUser;
+import dawahdevs.qa.api.entity.Lang;
+import dawahdevs.qa.api.entity.School;
+import dawahdevs.qa.api.entity.Seniority;
+import dawahdevs.qa.api.model.AllAuthorsResponse;
+import dawahdevs.qa.api.model.AllQuestionsResponse;
+import dawahdevs.qa.api.model.AllUsersResponse;
+import dawahdevs.qa.api.model.Author;
+import dawahdevs.qa.api.model.AuthorByIdResponse;
+import dawahdevs.qa.api.model.Question;
+import dawahdevs.qa.api.model.QuestionByIdResponse;
+import dawahdevs.qa.api.model.User;
+import dawahdevs.qa.api.model.UserByIdResponse;
+import lombok.AllArgsConstructor;
+
+@Component
+@AllArgsConstructor
+public class QaApiTransformer {
```

Comment:
I will reduce code with generic Util methods in the future, because of the way data is returned from the backend, with question/answer/user/author data all in one body... seems sloppy transform each part of the payload in a different class...

As for the manager, code is so simplified that the Manager class will not be overloaded or hard to follow.

Sadly with the way our backend is giving us data we are forced to "look" monolithic sometimes but I def see you point!